common/djangoapps/student/views.py

```diff
@@ -147,6 +148,12 @@
 # used to announce a registration
 REGISTER_USER = Signal(providing_args=["user", "registration"])
 
+if django.VERSION[0] == 1 and django.VERSION[1] < 10:
```

Comment:
You should put the shim TODO here so we can find it with the others later. We'd agreed on:

```
# TODO: Remove Django 1.11 upgrade shim
# SHIM: Description of the shim's purpose...
```